tasks/revision.js

```diff
@@ -1,78 +1,92 @@
 import { isDevelopment } from '../lib/env';
+
 import log from 'fancy-log';
 import config from '../lib/config';
-import gulp from 'gulp';
 import pump from 'pump';
-import runSequence from 'run-sequence';
-
 import rev from 'gulp-rev';
 import revDeleteOriginal from 'gulp-rev-delete-original';
 import revReplace from 'gulp-rev-replace';
+import { src, dest, task, parallel, series } from 'gulp';
 
-const manifestDir = config.get('tasks.revision') &&
+const MANIFEST_DIR = config.get('tasks.revision') &&
   config.get('tasks.revision.manifest.directory') ?
   config.get('tasks.revision.manifest.directory') : config.get('paths.dist');
-const manifestFile = config.get('tasks.revision') &&
+
+const MANIFEST_FILE = config.get('tasks.revision') &&
   config.get('tasks.revision.manifest.file') ?
   config.get('tasks.revision.manifest.file') : 'assets.json';
```

Comment:
Klein dingetje dat me toevallig opvalt bij het scannen (hierboven ook zie ik), maar ik zou de conditional op één regel clumpen en dan het vraagteken en dubbele punt op separate regels breken. 
Dan heb je een exclusieve regel voor de clausule en twee regels voor iedere tak.

Reply:
Ha, dit is inmiddels al vrij oude code, en onze standaard veranderen/verbeteren constant. Pas 'm gelijk ff aan; had alleen de constants even aangepakt, wat erachter kwam had ook meteen meegekund 👍